src/store/modules/ui.store.js

```diff
@@ -289,7 +289,7 @@ export default {
             state.fullscreenMode = mode
         },
         setEmbeddedMode(state, { embed }) {
```

Comment:
It would make sense to reflect the change `embedded` => `embed` also in the action/mutation name

Reply:
Done